src/main/java/org/dependencytrack/util/NotificationUtil.java

```diff
@@ -378,6 +403,23 @@ public static JsonObject toJson(final NewVulnerabilityIdentified vo) {
         return builder.build();
     }
 
+    public static JsonObject toJson(final VulnerabilityUpdate vo) {
+        final JsonObjectBuilder builder = Json.createObjectBuilder();
+        if (vo.getVulnerabilityUpdateDiff() != null) {
+            builder.add("old severity", vo.getVulnerabilityUpdateDiff().getOldSeverity().toString());
+        }
```

Comment:
Thinking of future use-cases here, where perhaps we would want to capture more than just a change in severity, WDYT about having an `old` and `new` object instead? This would give us the flexibility to add more properties of interest over time, e.g. CVSS or EPSS scores.

Looking at popular solutions for Change Data Capture, like Debezium, they typically do this, too: https://debezium.io/documentation/reference/stable/connectors/mysql.html#mysql-update-events

I'd imagine something like:

```json
{
  "component": {
    ...
  },
  "project": {
    ...
  },
  "vulnerability": {
    "old": {
      "severity": "MEDIUM"
    },
    "new": {
      "severity": "HIGH"
    }
}
```

Where `old` and `new` may just as well be `before` and `after`.

From the DT perspective, this form of communicating changes could be re-used in other notification types as well. WDYT?

cc @Mvld3r